### PR TITLE
add extension md to use proper markdown display

### PR DESCRIPTION
**Pull request by cove9988**

### CODE REVIEW

File: postgis/package/README.gppkg.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+
 ======================================
 
 To build postgis gppkg, we first need to download the third-party libraries from
 lvy repo by using make sync_tools provided by gpdb.
+
 Usage:
-	1. located to gpdb/gpAux folder
-	2. run make sync_tools [BLD_ARCH="rhel5_x86_64"]
+- 1. located to gpdb/gpAux folder
```
Comment (kaknikhil) [inline postgis/package/README.gppkg.md]: With the addition of `-`, we don't need the numbers 1. and 2.

Comment (kaknikhil) [reply postgis/package/README.gppkg.md]: @cove9988 can you make this change so that I can merge the PR ?

Comment (kaknikhil) [reply postgis/package/README.gppkg.md]: Thanks. 
Can you also squash all the "UPDATE" commits into one and write a brief commit message as well. 
I can also take care of the squashing if that's more convenient for you.

**Comment by cove9988** [comment]
Yes, Could you please squashing all the commits into one. Many thanks

**Pull request merged**